src/main/controllers/GeneralCorrespondenceListController.ts

```diff
@@ -0,0 +1,40 @@
+import { Response } from 'express';
+
+import { AppRequest } from '../definitions/appRequest';
+import { NotificationSubjects, TranslationKeys } from '../definitions/constants';
+import { FormContent } from '../definitions/form';
+import { AnyRecord } from '../definitions/util-types';
+
+import { getPageContent } from './helpers/FormHelpers';
+import { updateGeneralCorrespondenceRedirectLinksAndStatus } from './helpers/GeneralCorrespondenceHelper';
+
+export class GeneralCorrespondenceListController {
+  public get = async (req: AppRequest, res: Response): Promise<void> => {
+    const userCase = req.session?.userCase;
+
+    const translations: AnyRecord = {
+      ...req.t(TranslationKeys.CONTACT_THE_TRIBUNAL, { returnObjects: true }),
+      ...req.t(TranslationKeys.CITIZEN_HUB, { returnObjects: true }),
+      ...req.t(TranslationKeys.GENERAL_CORRESPONDENCE_LIST, { returnObjects: true }),
+    };
+
+    const content = getPageContent(req, <FormContent>{}, [
+      TranslationKeys.COMMON,
+      TranslationKeys.GENERAL_CORRESPONDENCE_LIST,
+    ]);
+
+    let correspondences;
+    if (userCase.sendNotificationCollection?.length > 0) {
```

Comment:
Can just check for ?.length

Reply:
I think this one should be with > 0 because it then calls the function with foreAch and in foreach I call properties on each object in the list, and if list is empty it could throw **TypeError: Cannot read property**

Reply:
But if you get the length and it's 0 then it's falsy so it inherently tests it to be bigger than 0.